setup/utils/build_protos.py

```diff
@@ -40,13 +42,13 @@ def clean(path):
     """
     # Start running from one directory above the directory which is found by
     # this scripts's location as __file__.
-    logging.info("Cleaning out compiled protos.")
+    LOGGER.info("Cleaning out compiled protos.")
     cwd = os.path.dirname(path)
 
     # Find all the .proto files.
     for (root, dirs, files) in os.walk(cwd):
         if is_grpc_service_dir(files):
-            logging.info('Skipping grpc service directory: %s', root)
+            LOGGER.info('Skipping grpc service directory: %s', root)
```

Comment:
Is there a reason that clean doesn't clean up the service directories?

Reply:
This I'm not sure about. @jiyunyao do you know why we don't clean service directories of built protos?